idx/memory/stats.go

```diff
@@ -7,9 +7,4 @@ var (
 	// a corruption has been detected in one of the internal index structures
 	// each time this happens, an error is logged with more details.
 	corruptIndex = stats.NewCounter32("recovered_errors.idx.memory.corrupt-index")
-
-	// metric recovered_errors.idx.memory.invalid-tag is how many times
-	// an invalid tag for a metric is encountered.
-	// each time this happens, an error is logged with more details.
-	invalidTag = stats.NewCounter32("recovered_errors.idx.memory.invalid-tag")
```

Comment:
`metrics.md` needs to be updated if this gets removed

Reply:
I added it back because I needed it in other places in memory. Originally it was moved up to idx, but I need it in both places. I will take a look at metrics.md and get it sorted out

Reply:
metrics.md is auto generated using the scripts